rxandroidble/src/main/java/com/polidea/rxandroidble2/scan/ScanResult.java

```diff
@@ -13,15 +13,17 @@ public class ScanResult {
     private final ScanCallbackType callbackType;
     private final ScanRecord scanRecord;
     private final IsConnectable isConnectable;
+    private final Integer advertisingSid;
 
     public ScanResult(RxBleDevice bleDevice, int rssi, long timestampNanos, ScanCallbackType callbackType,
-                      ScanRecord scanRecord, IsConnectable isConnectable) {
+                      ScanRecord scanRecord, IsConnectable isConnectable, Integer advertisingSid) {
```

Comment:
could we copy the constructor (telescope) and keep the original to not break the API? we could add `@RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)`